WORKSPACE

```diff
@@ -260,3 +260,30 @@ http_archive(
 load("@build_bazel_rules_nodejs//toolchains/esbuild:esbuild_repositories.bzl", "esbuild_repositories")
 
 esbuild_repositories(npm_repository = "npm")
+
+# Web testing
+
+http_archive(
+    name = "io_bazel_rules_webtesting",
+    sha256 = "e9abb7658b6a129740c0b3ef6f5a2370864e102a5ba5ffca2cea565829ed825a",
+    urls = [
+        "https://github.com/bazelbuild/rules_webtesting/releases/download/0.3.5/rules_webtesting.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_webtesting//web:repositories.bzl", "web_test_repositories")
+
+web_test_repositories()
+
+load("@io_bazel_rules_webtesting//web/versioned:browsers-0.3.3.bzl", "browser_repositories")
+
+browser_repositories(chromium = True)
+
+load("@io_bazel_rules_webtesting//web:go_repositories.bzl", web_test_go_repositories = "go_repositories")
+
+web_test_go_repositories()
+
+buildbuddy(
+    name = "buildbuddy_webdriver_toolchain",
+    docker_image = "docker://gcr.io/flame-public/rbe-webdriver@sha256:d7cf2332429dc2fca3fea127e18683129e44061ff04734bdc06392238ee6c1d7",
```

Comment:
I couldn't get that image to work since it doesn't include GCC.

Reply:
I'm not sure why GCC would be needed if the docker image is just used for running the test?

Also, if you're using the buildbuddy toolchain - you shouldn't need GCC, just clang.

Reply:
Thanks for catching this -- The GCC issue was probably a misconfiguration on my part since I can no longer repro the issue. Looks like it works just fine with this RBE image :+1: Updated to remove the Dockerfile from this PR and use this webtesting image for `exec_properties`